### PR TITLE
Rename page/file for Generic IP camera

### LINKED ISSUE #10486: 📚 Page/File name for Generic IP Camera
After reviewing PR #10484 I noticed that some integrations actually have strange meaningless names or are too global, so there is no structure.

In this case the **[Generic IP Camera](https://github.com/home-assistant/home-assistant.io/blob/next/source/_components/generic.markdown)**
It has the name `general.markdown`, I suggest something like `generic_ip_camera.markdown`.

**Checklist:**
- [ ] Make PR's against the `current` branch
- [ ] Rename page
- [ ] Add redirect from old to new page in the [_redirects file](https://github.com/home-assistant/home-assistant.io/blob/current/source/_redirects)
- [ ] Adjust all current links to old URL

### PR DESCRIPTION
**Pull request by thatguynikita**
**Description:**

This PR references changes required for  #10486 

- [x] Make PR's against the `current` branch
- [x] Rename page
- [x] Add redirect from old to new page in the [_redirects file](https://github.com/home-assistant/home-assistant.io/blob/current/source/_redirects)
- [x] Adjust all current links to old URL

Closes #10486 

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
The redirects file needs to be adjusted so that visitors to the old URL still go to the correct page.
I've added the checklist from the issue to this PR.

File: source/_redirects [hunk 1]
```diff
@@ -218,7 +218,7 @@
 /components/camera.doorbird /components/doorbird#camera
 /components/camera.familyhub /components/familyhub
 /components/camera.foscam /components/foscam
-/components/camera.generic /components/generic
+/components/camera.generic /components/generic_ip_camera
```
Comment (thatguynikita) [inline source/_redirects:221]: Thanks, Franck.
I thought just updating the redirects file for _/components/camera.generic_ should've been enough.
Or was I supposed to add a new redirect entry from _/components/generic_ to _/components/generic_ip_camera_ instead?

Comment (frenck) [reply source/_redirects:221]: Both! 😄

Comment (thatguynikita) [reply source/_redirects:221]: Thank you! I hope I've got it right this time.

(Apologies for any misunderstanding - first PR ever, very nervous 😄)

Comment (frenck) [reply source/_redirects:221]: Don't be nervous! We are all people and we all had the same experience once.
So don't worry! It always makes me happy to see somebody step up and joining the open-source game, that sir, deserves a medal 🏅

**Review by frenck — APPROVED** [review]
Looking good sir! Thank you so much for your contribution! 🥇 
Feel free to create more PR's in the future 👍 

Happy Hacktoberfest 🎉

File: source/_redirects (hunk 1 above)
Comment (thatguynikita) [reply source/_redirects:221]: Franck, thanks a lot!🤗 It means a great deal for me as a new-comer. I'm so glad to get introduced to the open-source community via this project. 

P.S. I'll save and print out this PR for getting back to it at any time of self-doubt 🙃

**Pull request merged**